packages/donations-app/app/components/project/projectController.js

```diff
@@ -29,7 +29,8 @@ angular.module('aliceApp')
       // myStory.details with an array of 'paragraph' objects
       let parseRegex = /<p>([^<]*)<\/p\>\s*/;
```

Comment:
Do we still need this regexp?

Reply:
We still need this to keep the overlay modal on the current appeal page modular